ansible/group_vars/all

```diff
@@ -16,6 +16,7 @@ runtimesManifest:
     default: true
   python:
   - kind: "python"
```

Comment:
`kind:python` must be python2action because all pre-existing python actions are of this kind and hence may not work with python3. Hence to preserve the behavior, we have to map `python` to 2, not 3.

so the changes are:

```
kind: python,         image_name: python2Action
kind: python:2,       image_name: python2Action
kind: python:3,       image_name: python3Action, default: true
```